torch/nn/utils/rnn.py

```diff
@@ -318,15 +319,26 @@ def pad_sequence(sequences, batch_first=False, padding_value=0):
         out_dims = (len(sequences), max_len) + trailing_dims
     else:
         out_dims = (max_len, len(sequences)) + trailing_dims
-
     out_tensor = sequences[0].data.new(*out_dims).fill_(padding_value)
-    for i, tensor in enumerate(sequences):
-        length = tensor.size(0)
-        # use index notation to prevent duplicate references to the tensor
-        if batch_first:
-            out_tensor[i, :length, ...] = tensor
-        else:
-            out_tensor[:length, i, ...] = tensor
+
+    if not isinstance(padding_type, str):
+        raise TypeError("Expected padding_type to be str type but {} found".format(type(padding_type)))
```

Comment:
I think this is unnecessary. There's an error case below for other values too

Reply:
Oop, fixed